### ISSUE TITLE
issue

### ISSUE THREAD

**Issue by Youqiang-Gui** [fordevoted/UtilityIR#5]
I downloaded the All-Weather dataset file you uploaded. Please check your dataset carefully. I would like to remind you that in your rainstreak dataset, the training set contains the test set. There are 750 images starting from im_0301_s80_a04.png in total. They exist in both the test machine and the training set. I think this is wrong. The same is true for the raindrop dataset. The training set also contains test set images.

**Comment by TanMIMI** [comment]
I also have some questions about the results presented in your paper. Specifically, I noticed a significant difference in the statistics for the same model when compared to the results from another paper, despite both being trained on the same dataset. Below are the results from your paper:
![image](https://github.com/user-attachments/assets/515f0cf8-e98c-413e-9a31-6861eade2f13)
And here are the results from the other paper:
![image](https://github.com/user-attachments/assets/6e81abde-404a-4a7e-9ab7-e2475f2cabea)
The disparity between the results is quite large. Could you provide some insights into what might account for this difference?

**Comment by fordevoted** [comment]
> I downloaded the All-Weather dataset file you uploaded. Please check your dataset carefully. I would like to remind you that in your rainstreak dataset, the training set contains the test set. There are 750 images starting from im_0301_s80_a04.png in total. They exist in both the test machine and the training set. I think this is wrong. The same is true for the raindrop dataset. The training set also contains test set images.

Thank you for your kindly reminder. We remove these images from the training set and retrain the model. The results still show superiority compared to other methods and close to the score show in the paper. If you still encounter any difficulties for training, feel free to let me know. Thanks.

**Comment by fordevoted** [comment]
> I also have some questions about the results presented in your paper. Specifically, I noticed a significant difference in the statistics for the same model when compared to the results from another paper, despite both being trained on the same dataset. Below are the results from your paper:
> ![image](https://github.com/user-attachments/assets/515f0cf8-e98c-413e-9a31-6861eade2f13)
> And here are the results from the other paper:
> ![image](https://github.com/user-attachments/assets/6e81abde-404a-4a7e-9ab7-e2475f2cabea)
> The disparity between the results is quite large. Could you provide some insights into what might account for this difference?

We evaluate PSNR/SSIM for Y channel in YCbCr color space. The other paper you provided (WGWSNet I think) might evaluate in RGB color space. Implementation details can also make the result different, and it can be figured out in the source code.
